operator/controllers/config/constants.go

```diff
@@ -195,4 +194,11 @@ const (
 
 	ErrorForbidden    = "403: Forbidden"
 	ErrorUnauthorized = "401: Unauthorized"
+
+	ENVCluster_ConfigurationType          = "ClusterConfigurationType"
```

Comment:
Some good to have variable renaming along with other changes:
It will be good to make var names better from here and nodeserver.go e.g. 
1. remove underscores from var names and use [mixedCaps](https://go.dev/doc/effective_go#mixed-caps)
2. `ENVCluster_CNSAOperatorNamespace` and `ENVCluster_CNSAOperatorDeploymentName` are not being used as env vars so better to remove `ENVCluster_` from these